inc/namespace.php

```diff
@@ -51,6 +51,12 @@ function bootstrap() {
 		add_action( 'login_header', __NAMESPACE__ . '\\add_login_logo' );
 	}
 
+	if ( isset( $config['favicon-icon'] ) ) {
```

Comment:
Should we allow devs to override the behavior by using `! empty( .. )` instead ? So they'd be able to use false and empty strings as values.

Reply:
Actually, could we use the filter in `get_site_icon_url` instead ? So we don't have to output our own markup, and ensure consistency with WordPress markup ?

Ref: https://developer.wordpress.org/reference/functions/get_site_icon_url/

Reply:
What if I were to change it to use `has_site_icon` instead so if the site Icon is set already, then those functions don't even run yet.

Ref: https://developer.wordpress.org/reference/functions/has_site_icon/

With that change it would eliminate the functions from running unnecessary. As the `favicon-icon` has a default setting so it would always essentially have a value.

Reply:
Hmmm, possibly both ? Checking if there's a valid config value OR a valid setting value as well. Then hooking to the filter.